apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -471,44 +471,31 @@ export const getMultipleChoiceSectionSummary = (state) => {
 /**
  * Takes in an array of objects {answerText: '', correct: true/false} and
  * returns the corresponding letters to the options with the correct answers.
- *
- * TODO(caleybrock): Add letter options to response from the server so they are
- * consistent with the structure, but for now look up letter in this array.
- * If this code is left client side, it needs tests.
- *
  * Ex - [{correct: false}, {correct: true}] --> returns 'B'
  */
-const getCorrectAnswer = (answerArr) => {
+export const getCorrectAnswer = (answerArr) => {
```

Comment:
I actually prefer not to use arrow functions for top-level exports, where there's no `this` value to capture.  Mostly it's a style thing, but using a regular function declaration also binds the function name to the function object, which makes for more readable stack traces in some browsers (I think Chrome is clever enough that this doesn't matter anymore).

```js
export function getCorrectAnswer(answerArr) {
```